content/guides/exporters/supported-exporters/Node.js/stackdriver-stats.md

```diff
@@ -44,7 +44,7 @@ export GOOGLE_APPLICATION_CREDENTIALS=path/to/your/credential.json
 
 * Create the exporters in code
 
-#### Creating the exporter in code
+## Creating the exporter
```

Comment:
Maybe just remove this tab? (This tab is removed in stackdriver-trace.md)

Reply:
Roger that, thanks!